client/dogs/FormPage.js

```diff
@@ -316,6 +318,8 @@ type State = {
   rabiesIssued: string,
   rabiesExpire: string,
   comments: string,
+  rabiesCert: Array<Object>,
```

Comment:
Thanks for the reference. I'd think `Array<File>` would be more appropriate too, but when I `console.log(typeof file)` I'm getting back `object`.